src/mmw/js/src/modeling/tr55/models.js

```diff
@@ -28,6 +28,10 @@ var AoiVolumeModel = Backbone.Model.extend({
     adjust: function(depth) {
         // Adjusted runoff is depth (cm) -> meters * the AoI area (m2)
         return (depth / 100) * this.get('aoiArea');
+    },
+
+    getLoadingRate: function(load) {
+        return load / (this.get('aoiArea') / 10000);
```

Comment:
Consider adding a comment mentioning units, and explaining why we need to divide by 10000.